certbot/tests/test_util.py

```diff
@@ -111,3 +115,46 @@ def skip_unless(condition, reason):  # pragma: no cover
         return lambda cls: cls
     else:
         return lambda cls: None
+
+
+def make_lineage(self, testfile):
```

Comment:
@ohemorange, a few things for posterity, `make_lineage` creates as lineage based on any of the configuration files in `certbot/tests/testdata/*.conf`. While `BaseRenewableCertTest` makes a lineage (correct word?) on the fly, based on some parameters in its class definition. It would be nice to remove the dependency of `BaseRenewableCertTest` since it is imported from another test file.

After a few attempts I could not get the `DuplicativeCertsTest` tests to work using  `make_lineage`.  So I'll punt for now and make another issue.